src/scss/_docs.scss

```diff
@@ -18,6 +18,12 @@
     font-size: 14px;
   }
 
+  img.screenshot-partial {
+    border-radius: 4px;
+    border: #E0E6ED 1px solid;
+    margin: 10px 0 10px 0;
```

Comment:
Let's make this 16 and just put `16px 0`. first number is top and bottom second is top and left same value.